src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/variable_speed_compressor_train_common_shaft_multiple_streams_and_pressures.py

```diff
@@ -879,7 +892,7 @@ def calculate_compressor_train_given_rate_ps_speed(
             )
 
             mass_rate_previous_stage_kg_per_hour = mass_rate_this_stage_kg_per_hour
-
+            self.fluid_to_recirculate_in_stage[stage_number] = inlet_stream
```

Comment:
possibly to replace set and get to this member variable using methods to more clearly indicate that we are adding "leftover rate" in one timestep that is "captured" in next timestep? e.g.

`set_recirculate_fluid_in_next_timestep(stage, stream)`

and

`get_fluid_to_recirculate(stage)`

or some other names that make more sense? Does it make sense?

Reply:
But is it actually "next timestep"? Isnt it "next recirculation round" or whatever name is being used by experts? to not be confused with "ecalc timesteps"?

Reply:
unless, I am wrong ;)

Reply:
> possibly to replace set and get to this member variable using methods to more clearly indicate that we are adding "leftover rate" in one timestep that is "captured" in next timestep? e.g.
> 
> `set_recirculate_fluid_in_next_timestep(stage, stream)`
> 
> and
> 
> `get_fluid_to_recirculate(stage)`
> 
> or some other names that make more sense? Does it make sense?

Yes, surely. Was a bit unsure how to approach it, but that sounds better.